gsl_projet/services.py

```diff
@@ -55,3 +109,14 @@ def get_ordering_arg(cls, ordering):
         }
 
         return ordering_map.get(ordering, None)
+
+    @classmethod
+    def compute_taux_from_montant(cls, projet: Projet, new_montant: float):
```

Comment:
```suggestion
    def compute_taux_from_montant(cls, projet: Projet, new_montant: float) -> Decimal:
```

(ça ne coûte pas grand-chose ici)